public/index.html

```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <meta name="image" property="og:image" content="./bookstore.png" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <!--
```

Comment:
- [ ] Nice Design :clap: but repetition is not a best practice in design because it can lead to user frustration and a negative user experience. Users expect to see new and relevant information as they navigate through our app or website, and encountering the same content repeatedly can be both tedious and confusing, I can see on your design we do have a `chapter17` which comes by default when adding the book in our store, I don't really get this I can see is not the same as on the design, I tried to inline them below for more visibility, kindly fix this to avoid confusion and user frustration :wink:  :point_down:  You should match your design to the one provided (original Design)
Your Design 
![Screenshot from 2023-03-04 15-15-06](https://user-images.githubusercontent.com/88894525/222904360-decfb327-b949-4018-aeb1-dfc45421e213.png)

Original Design 
![Screenshot from 2023-03-04 15-25-45](https://user-images.githubusercontent.com/88894525/222904404-a3f7791a-6925-4900-bd8e-28bf3f4050f5.png)
_Note: Styling is **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)_